commands/fun/trivia.js

```diff
@@ -0,0 +1,67 @@
+const Social = require(`${process.cwd()}/base/Social.js`);
+const Discord = require("discord.js");
+const snek = require("snekfetch");
+const B64 = require("base-64");
+class Trivia extends Social {
+  constructor(client) {
+    super(client, {
+      name: "trivia",
+      description: "Delivers a quiz with the correct answer awarding 10 points.",
+      category: "Fun",
+      usage: "trivia",
+      extended: "Ready to wrack your brain? ",
+      cooldown: 10,
+      aliases: ["quiz"],
+      botPerms: ["EMBED_LINKS"]
+    });
+  }
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+
+    const inf = await snek.get("https://opentdb.com/api.php?amount=10&category=9&difficulty=medium&type=multiple&encode=base64");
+    const ob = await inf.body;
+    const emb = new Discord.MessageEmbed();
+    var a;
+    var b;
+    var c;
+    var d;
+    var choice;
+    const questions = [];
+
+    ob.results[0].incorrect_answers.forEach(ob => questions.push(B64.decode(ob)));
+    questions.push(B64.decode(ob.results[0].correct_answer));
+    
+    a = await questions.random();
+    questions.splice(questions.indexOf(a), 1);
+    b = questions.random();
+    questions.splice(questions.indexOf(b), 1);
+    c = questions.random();
+    questions.splice(questions.indexOf(c), 1);
+    d = questions[0];
+
+    emb.setAuthor("Misaki Trivia", this.client.user.displayAvatarURL());
+    emb.setDescription(B64.decode(ob.results[0].question).replace(/â/g,"").replace(/â/g,""));
+    emb.setColor("#FC963B");
+    emb.addField("A", a);
+    emb.addField("B", b);
+    emb.addField("C", c);
+    emb.addField("D", d);
+
+    const question = await this.client.awaitReply(message, null, m => m.author.id === message.author.id, 60000, {embed:emb});
+    
+    if (question === false) return message.reply("you took too long to respond.");
+    if (question.toLowerCase() === "a") choice = a;
```

Comment:
This is incredibly unefficient, you're checking 4 ifs given that only one will be true (or none). Use `if/else`, or even better: [switch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch).

```javascript
switch(question.toLowerCase()) {
  case "a":
  case "b":
  case "c":
  case "d":
    choice = question.toLowerCase();
  default:
    return message.reply(...);
}
```

Consider that pattern for the next chained checks. However, it'd be much better for this case if you do:

```javascript
if (!/^(a|b|c|d)$/i.test(question)) return message.reply("that isn't a valid choice..");
choice = question.toLowerCase();
```